src/p5.dance.js

```diff
@@ -1126,7 +1147,11 @@ module.exports = class DanceParty {
 
   // Called when executing the AI block.
   ai(params) {
+    this.world.aiBlockCalled = true;
     console.log('handle AI:', params);
+    if (this.contextType === 'this.p5_.keyWentDown' && this.contextKey) {
```

Comment:
Nit: move this string into a constant?

Also, reading thru the PR description, the value for `aiBlockContextUserEventKey` is up/down/left/right, right? Could you throw a comment in here noting that, or include something in the variable name that indicates that this key isn't like a "key as in key/value", but rather a "key as in key pressed"?

Reply:
Good call! Just updated - thanks!